gym/bin/🏋️

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.push File.expand_path("../../lib", __FILE__)
+
+require "gym"
+require "gym/commands_generator"
+Gym::CommandsGenerator.start
```

Comment:
yep - is a copy of 💪 which is a copy of `/bin/gym` - not a fan of copy and 🍝 
but i think its worth the fun - besides the code is just a couple of lines.